pytorch_lightning/accelerators/accelerator.py

```diff
@@ -222,6 +219,18 @@ def __setstate__(self, d):
     def on_save(self, checkpoint):
         return checkpoint
 
+    @property
+    def rpc_enabled(self):
+        return self.ddp_plugin is not None and isinstance(self.ddp_plugin, RPCPlugin)
+
+    @property
+    def distributed_sampler_kwargs(self):
+        raise NotImplementedError
```

Comment:
Do we want a default implementation for that? I.e. returning an empty duct here as default. I don't think having a property raising NotImplementedError in a property is good practice.